package-res/resources/web/pentaho/visual/role/_doc/spec/UMappingAttribute.jsdoc

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A `spec.UMappingAttribute` represents the union of JS types that can be used to describe a
+ * [visual role mapping attribute]{@link pentaho.visual.role.MappingAttribute} instance.
+ *
+ * When an instance is described using _object form_,
+ * its specification is of type [spec.IMappingAttribute]{@link pentaho.visual.role.spec.IMappingAttribute}.
```

Comment:
Suggest:

its specification is of the type [spec.IMappingAttribute]{@link pentaho.visual.role.spec.IMappingAttribute}.